ci/build_docs.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -euo pipefail
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-dependency-file-generator \
+  --output conda \
+  --file_key docs \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee env.yaml
```

Comment:
For some reason, this is resolving to `cudatoolkit=11.5` even though we probably want 11.8.

Reply:
Same for Python -- we want 3.10, but got 3.9.

Reply:
This is probably because the custom job uses the "latest" container which is designated as CUDA 11.5, Python 3.9 for now. I'll resolve this conversation.